edx/edx-ora2
# Mattcarter/educator 5607

**TL;DR -** Rubric assessment sections had excessive vertical whitespace between them. 

JIRA: [EDUCATOR-5607](https://openedx.atlassian.net/browse/EDUCATOR-5607)

**What changed?**
- set margin-bottom to 0px for class

**Developer Checklist**
- [x] Bumped version number in setup.py and package.json

**Testing Instructions**
Test via devstack. 
 - Checkout this PR, and install it as the local ORA codebase.  
 - Create an ORA and configure it with a rubric 
 - Set 'Show Rubric During Response' to True
 - Navigate to the ORA and view as a learner. Expand the rubric sections.  There should be less vertical whitespace.
 - To confirm navigate the DOM and find the `<li>` element has an effective margin-bottom of 0px

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
